microsoft-edge/webview2/reference/javascript/webview.yml

```
@@ -0,0 +1,386 @@
+### YamlMime:TSType
+name: WebView
+uid: 'WebView2Script!WebView:class'
+package: WebView2Script!
+fullName: WebView
+summary: >-
+  `window.chrome.webview` is the class to access the WebView2-specific APIs that are available to the script running
+  within WebView2 Runtime.
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: class
+properties:
+  - name: hostObjects
+    uid: 'WebView2Script!WebView#hostObjects:member'
+    package: WebView2Script!
+    fullName: hostObjects
+    summary: >-
+      Contains asynchronous proxies for all host objects added via `CoreWebView2.AddHostObjectToScript` as well as
+      options to configure those proxies, and the container for synchronous proxies.
+
+
+      If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your host code then an asynchronous proxy
+      for `object` will be exposed in script as `chrome.webview.hostObjects.myObject`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'hostObjects: HostObjectsAsyncRoot;'
+      return:
+        type: '<xref uid="WebView2Script!HostObjectsAsyncRoot:class" />'
+        description: >-
+
+
+          #### Examples
+
+
+
+          For example, suppose you have a COM object with the following interface.
+
+
+          ```
+
+          [uuid(3a14c9c0-bc3e-453f-a314-4ce4a0ec81d8), object, local]
+
+          interface IHostObjectSample : IUnknown
+
+          {
+              // Demonstrate basic method call with some parameters and a return value.
+              HRESULT MethodWithParametersAndReturnValue([in] BSTR stringParameter, [in] INT integerParameter, [out, retval] BSTR* stringResult);
+
+              // Demonstrate getting and setting a property.
+              [propget] HRESULT Property([out, retval] BSTR* stringResult);
+              [propput] HRESULT Property([in] BSTR stringValue);
+
+              [propget] HRESULT IndexedProperty(INT index, [out, retval] BSTR * stringResult);
+              [propput] HRESULT IndexedProperty(INT index, [in] BSTR stringValue);
+
+              // Demonstrate native calling back into JavaScript.
+              HRESULT CallCallbackAsynchronously([in] IDispatch* callbackParameter);
+
+              // Demonstrate a property which uses Date types
+              [propget] HRESULT DateProperty([out, retval] DATE * dateResult);
+              [propput] HRESULT DateProperty([in] DATE dateValue);
+
+              // Creates a date object on the native side and sets the DateProperty to it.
+              HRESULT CreateNativeDate();
+          };
+
+          ```
+
+
+          Add an instance of this interface into your JavaScript with `AddHostObjectToScript`. In this case, name it
+          `sample`.
+
+
+          In native host app code:

```

```suggestion
          In the native host app code:
```
imo, thought better flow/context-setting by adding 'the'.